simbad/command_line/__init__.py

```diff
@@ -214,63 +219,76 @@ def _argparse_job_submission_options(p):
-    sg.add_argument("-morda_db", type=lambda x: is_valid_dir(sg, x), default=simbad.MORDA_MODELS, help="Path to local copy of the MoRDa database")
-    sg.add_argument("-max_morda_results", type=int, default=200, help="The maximum number of contaminant results to return")
+    sg.add_argument("-morda_db", type=lambda x: is_valid_dir(sg, x), default=simbad.MORDA_MODELS,
+                    help="Path to local copy of the MoRDa database")
+    sg.add_argument("-max_morda_results", type=int, default=200,
+                    help="The maximum number of contaminant results to return")
 
 
 def _argparse_lattice_options(p):
     """Lattice search specific options"""
     sg = p.add_argument_group("Lattice search specific options")
-    sg.add_argument("-latt_db", type=lambda x: is_valid_file(sg, x), default=simbad.LATTICE_DB, help="Path to local copy of the lattice database")
-    sg.add_argument("-max_lattice_results", type=int, default=20, help="The maximum number of lattice results to return")
+    sg.add_argument("-latt_db", type=lambda x: is_valid_file(sg, x), default=simbad.LATTICE_DB,
+                    help="Path to local copy of the lattice database")
+    sg.add_argument("-max_lattice_results", type=int, default=20,
+                    help="The maximum number of lattice results to return")
     sg.add_argument("-max_penalty_score", type=int, default=7, help="The maximum lattice penalty score allowed")
 
 
 def _argparse_rot_options(p):
     """Rotation search specific options"""
     sg = p.add_argument_group("AMORE Rotation search specific options")
-    sg.add_argument("-amore_exe", type=str, default=os.path.join(os.environ["CCP4"], "libexec", "amore-rs"), help="Path to amore executable")
-    sg.add_argument("-npic", type=int, default=50, help="Number of peaks to output from the translation function map for each orientation")
-    sg.add_argument("-min_solvent_content", type=int, default=30, help="The minimum solvent content present in the unit cell with the input model")
+    sg.add_argument("-amore_exe", type=str, default=os.path.join(os.environ["CCP4"], "libexec", "amore-rs" + EXE_EXT),
+                    help="Path to amore executable")
+    sg.add_argument("-npic", type=int, default=50,
+                    help="Number of peaks to output from the translation function map for each orientation")
+    sg.add_argument("-min_solvent_content", type=int, default=30,
+                    help="The minimum solvent content present in the unit cell with the input model")
     sg.add_argument("-pklim", type=float, default=0.5, help="Peak limit, output all peaks above <pklim>")
     sg.add_argument("-rotastep", type=float, default=1.0, help="Size of rotation step")
-    sg.add_argument("-rot_program", type=str, default="amore", choices=["amore", "phaser"], help="Program with which to perform rotation search")
+    sg.add_argument("-rot_program", type=str, default="phaser", choices=["amore", "phaser"],
```

Comment:
The idea of consistency does appeal to me but I think this would be overcomplicating things atm. I use the enum classes to specify paths/options in `mr/__init__.py`. I realised that I could use the keys in these classes to specify choices in argparse I would have to modify the enum class if I added in a new method. In the rotation search I'm not using an enum class atm so it isn't as convenient.